backend/coins/btc/electrum/client/client.go

```diff
@@ -308,26 +290,15 @@ func (client *ElectrumClient) TransactionGet(
 		txHash.String())
 }
 
-type electrumHeader struct {
-	// Provided by v1.4
-	BlockHeight int `json:"block_height"`
-	// Provided by v1.2
-	Height int `json:"height"`
```

Comment:
Of course. I naively trusted the comments but they are reversed. v1.4+ is just `height`. Earlier versions were `block_height`.

Reply:
Done